test/utils/helperFn.ts

```diff
@@ -191,3 +191,7 @@ export const getSelectInput = (combobox: Element) => {
 export function getSelectByName(name: string) {
   return getSelectInput(screen.getByRole('combobox', { name }))!;
 }
+
+export const includeRowSelection = (ids: GridRowId[]) => {
```

Comment:
Could this be useful as a public export too?
For example, it could be used as a solution in migration guide diff (and also in the codemod probably).

```diff
-const [rowSelectionModel, setRSM] = React.useState([1, 2, 3]);
+const [rowSelectionModel, setRSM] = React.useState(includeRowSelection([1, 2, 3]));
```